README.md

```diff
@@ -37,16 +35,14 @@ pipeline:
       image: public.ecr.aws/gumgum-verity/oss/astro-ap-build:0.2.0
       <<: *astrocloud_env
       commands:
-        - docker login images.astronomer.cloud -u $${ASTRONOMER_KEY_ID} -p $${ASTRONOMER_KEY_SECRET}
+        - echo ${ASTRONOMER_API_TOKEN} | docker login images.astronomer.cloud -u _json_key --password-stdin
```

Comment:
@Lowess just this part, I tried to execute the command docker login with the token secret in several ways but with no success, Do you have any idea on how to do it? We need to specify the username but we don't have one, I was trying to use the token id but you can never see it in the Astronomer GUI.

Reply:
This part does not need to change, please roll back. Astrocloud has changed the way the deployment is done not the way the docker image is pushed to their registry. Docker registry login should still continue using key id and key secret